modules/sdk-coin-dot/src/lib/iface.ts

```diff
@@ -179,7 +179,7 @@ export type StakeArgsPayeeRaw = { controller?: null; stash?: null; staked?: null
  */
 export interface StakeArgs {
   value: string;
-  controller?: { id: string };
+  controller: { id: string };
```

Comment:
Undocumented breaking change. Need a BREAKING CHANGE: <comment> in the commit footer.

Reply:
this is actually reverting the previous change https://github.com/BitGo/BitGoJS/commit/a6f6a0d96afdb663a7a096c4eb2caf9ca1649d63

Reply:
i'll keep that in mind for future breaking changes